framer/Version.coffee.template

```diff
@@ -0,0 +1,7 @@
+# This gets automatically populated bij gulp
+
+exports.date = <%= date %>
```

Comment:
Hmm yeah good one, maybe for hashing. But I can't really think of something this will break right now.